Samples/Client/SwiftClient/SwiftClient/VideoStreamViewController.swift

```diff
@@ -0,0 +1,668 @@
+import UIKit
+import CoreMotion
+import WebRTC
+
+class VideoStreamViewController: UIViewController {
+    @IBOutlet weak var connectDisconnectButton: UIButton!
+    @IBOutlet weak var pickerView: UIPickerView!
+    var mathMatrix = MathMatrix()
+    var request: URLSessionDataTask!
+    var hangingGet: URLSessionDataTask!
+    var otherPeers = [Int: String]()
+    var myId: Int!
+    var heartBeatTimer: Timer!
+    var heartBeatTimerIsRunning = false
+    let heartbeatIntervalInSecs = 5
+    let server = Config.signalingServer
+    let localName = "ios_client"
+    var peerConnection: RTCPeerConnection!
+    var peerConnectionFactory = RTCPeerConnectionFactory()
+    var newIceCandidatePeerId = -1
+    static let peerListDisplayInitialMessage = "Select peer to join"
```

Comment:
nit: can we logically group these? ie: statics, then @IBOutlets, then let, then var

Reply:
Yes, will do